tests/test_acasclient.py

```diff
@@ -437,6 +437,16 @@ def create_and_connect_backdoor_user(self, username = None, password = None, pre
         user_client = acasclient.client(user_creds)
         return user_client
 
+    def verify_file_and_content_equal(self, content, file):
```

Comment:
`:nit` If the function name is `_file_and_content_` then I would expect the first argument to be the file path and the second argument as the content.